code/src/WijmoProvider/Features/ColumnFilter.ts

```diff
@@ -108,6 +108,62 @@ namespace WijmoProvider.Feature {
             this.setState(this._enabled);
         }
 
+        public byCondition(
+            columnId: string,
+            values: OSFramework.OSStructure.FilterCondition[]
+        ): void {
+            const column = this._grid.getColumn(columnId);
+            if (column) {
+                const columnFilter = this._filter.getColumnFilter(
```

Comment:
This line will work even if the column has the filter disabled, right?

Reply:
Yes!